lib/service/rpmsg/rpc/rpmsg_rpc_client.c

```diff
@@ -6,6 +6,7 @@
  */
 
 #include <openamp/rpmsg_rpc_client_server.h>
+#include <errno.h>
```

Comment:
We usually follow alphabetical ordering for the` #include` declarations.
Please place it first in this file and also in rpmsg_rpc_server.c.

Else LGTM

Reply:
Thanks. Fixed.